### ISSUE TITLE
Support for pagination

### ISSUE THREAD

**Issue by hedder** [jakubrohleder/angular-jsonapi#24]
Of.spec has a statement regarding pagination

> A server MAY choose to limit the number of resources returned in a response to a subset ("page") of the whole set available.
> 
> A server MAY provide links to traverse a paginated data set ("pagination links").
> 
> Pagination links MUST appear in the links object that corresponds to a collection. To paginate the primary data, supply pagination links in the top-level links object. To paginate an included collection returned in a compound document, supply pagination links in the corresponding links object.
> 
> The following keys MUST be used for pagination links:
> 
> first: the first page of data
> last: the last page of data
> prev: the previous page of data
> next: the next page of data

<a href="http://jsonapi.org/format/#fetching-pagination">here</a>

Do you have plans to support pages?

**Comment by jakubrohleder** [comment]
Certainly! I'm back from my journey and this is the next point on my roadmap. 

I'll prepare the paginated object API example today and if it suits you i'll add it in the next release (probably by the end of a week).

**Comment by LavaToaster** [comment]
@jakubrohleder Do you have any ETA?

**Comment by pablorsk** [comment]
I'm interested too :)
